config/locales/en.yml

```diff
@@ -959,6 +962,106 @@ en:
+    create:
+      successful_report: Your report has been registered sucessfully
+      provide_details: Please provide the required details
+    update:
+      new_report: Your report been registered sucessfully
+      successful_update: Your report has been updated successfully
+      provide_details: Please provide the required details
+    new:
+      details: Please provide some more details about the problem (required).
+      select: "Select a reason for your report:"
+      disclaimer:
+        intro: "Before sending your report to the site moderators, please ensure that:"
+        not_just_mistake: You are certain that the problem is not just a mistake
+        unable_to_fix: You are unable to fix the problem yourself or with the help of your fellow community members
+        resolve_with_user: You have already tried to resolve the problem with the user concerned
+    show:
+      title: "%{status} Issue #%{issue_id}"
+      reports:
+        zero: No reports
+        one: 1 report
+        other: "%{count} reports"
+      report_created_at: "First reported at %{datetime}"
+      last_resolved_at: "Last resolved at %{datetime}"
+      last_updated_at: "Last updated at %{datetime} by %{displayname}"
+      reports_of_this_issue: Reports of this Issue
+      read_reports: Read Reports
+      new_reports: New Reports
+      other_issues_against_this_user: Other issues against this user
+      no_other_issues: No other issues against this user.
+      comments_on_this_issue: Comments on this issue
+      comments:
+        reassign_param: Reassign Issue?
+    comment:
+      provide_details: Please provide the required details
+      comment_created: Your comment was successfully created
+    comments:
+      created_at: "On %{datetime}"
+    reports:
+      category: "Category: %{category}"
+      updated_at: "On %{datetime}"
+      reported_by_html: "Reported by <a href=\"%{user_url}\">%{user_name}</a>"
+    resolved: Issue status has been set to 'Resolved'
+    ignored: Issue status has been set to 'Ignored'
+    reopened: Issue status has been set to 'Open'
+    states:
+      ignored: Ignored
+      open: Open
+      resolved: Resolved
+  reports:
+    new:
+      title_html: "Report %{link}"
+    categories:
+      DiaryEntry:
+        spam: This Diary Entry is/contains spam
+        offensive: This Diary Entry is obscene/offensive
+        threat: This Diary Entry contains a threat
+        other: Other
+      DiaryComment:
+        spam: This Diary Comment is/contains spam
+        offensive: This Diary Comment is obscene/offensive
+        threat: This Diary Comment contains a threat
+        other: Other
+      User:
+        spam: This User profile is/contains spam
+        offensive: This User profile is obscene/offensive
+        threat: This User profile contains a threat
+        vandal: This User is a vandal
+        other: Other
+      Note:
+        spam: This note is spam
+        vandalism: This note is vandalism
```

Comment:
Not sure how a note can be vandalism? What exactly is it vandalising?

Reply:
Maybe if it's intentionally misleading for non-spam reasons (just "for the lulz" stating "There is a post box here")?

Reply:
But that would only become vandalism if somebody went ahead and added it to the map without validating it?